src/main/resources/static/auth/index.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <title>LoginPage</title>
+    <meta name="description" content=""/>
+    <meta name="viewport" content="width=device-width"/>
+    <style>
+        body {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            height: 100vh;
+            margin: 0;
+        }
+
+        .container {
+            text-align: center;
+        }
+        img{
+            width: 100px;
+            height: 100px;
+        }
+    </style>
+</head>
+<body>
+<div class="container">
+    <div class="container unauthenticated">
+        <h1>Sign in </h1>
+        <a href="/oauth2/authorization/github">
+        <img src="https://github.com/fluidicon.png" alt="GitHub Logo">
+    </a>
+    </div>
+</div>
+</body>
```

Comment:
No semantic HTML? 😜

Reply:
Additionally, I wouldn't link an image from another site like this - you never know when they might change the path to it: that would look bad on our site!